libvncclient/sockets.c

```diff
@@ -862,6 +866,11 @@ PrintInHex(char *buf, int len)
   fflush(stderr);
 }
 
+void CancelConnection(rfbClient* client)
```

Comment:
Can this also cancel a running connection, i.e. act as a disconnect? If so, please document it and it it to the header file.